pkg/broker/handler/processors/filter/processor.go

```diff
@@ -55,10 +57,24 @@ func (p *Processor) Process(ctx context.Context, event *event.Event) error {
 		return nil
 	}
 
+	trigger := types.NamespacedName{
+		Namespace: target.Namespace,
+		Name:      target.Name,
+	}
+	var span *trace.Span
 	if dt, ok := extensions.GetDistributedTracingExtension(*event); ok {
-		var span *trace.Span
-		ctx, span = dt.StartChildSpan(ctx, fmt.Sprintf("trigger:%s.%s", target.Name, target.Namespace))
-		defer span.End()
+		ctx, span = dt.StartChildSpan(ctx, kntracing.TriggerMessagingDestination(trigger))
+	} else {
+		ctx, span = trace.StartSpan(ctx, kntracing.TriggerMessagingDestination(trigger))
+	}
+	defer span.End()
+	if span.IsRecordingEvents() {
+		span.AddAttributes(
+			kntracing.MessagingSystemAttribute,
+			tracing.PubSubProtocolAttribute,
+			kntracing.TriggerMessagingDestinationAttribute(trigger),
```

Comment:
Should this include any form of type information? As-is, I can't tell the difference between different objects that have the same name and namespace (e.g. K8s service vs Broker vs KafkaChannel).

Reply:
messaging.destination is prefixed by either broker or trigger and will contain the namespaced name.